wrappers/ortools.rb

```diff
@@ -590,8 +590,11 @@ def parse_output(vrp, output)
           service = vrp.services.find{ |s| s.id == service_id }
           {
             original_service_id: service.original_id,
+            pickup_shipment_id: service.type == :pickup && service.original_id,
+            delivery_shipment_id: service.type == :delivery && service.original_id,
```

Comment:
Don't we need such a precision in vroom.rb?

Reply:
It already exists in vroom wrapper, it was only missing here in the unassigned jobs of ortools